scripts/docker_build.sh

```diff
@@ -26,10 +26,7 @@ IMAGES["fedora:34"]="1"
 build_image() {
     BASE_IMAGE=${1}
     IMAGE_TAG=$(echo ${BASE_IMAGE} | tr -s :/ - ) # replace colons and slashes with hyphens
-
-    # pull the rust version out of ../rust-toolchain.toml to keep things synced
-    RUST_TOOLCHAIN_FILE="$SCRIPT_DIR/../rust-toolchain.toml"
-    RUST_VER=$($SCRIPT_DIR/query_toml.py toolchain.channel $RUST_TOOLCHAIN_FILE)
+    RUST_VER=stable
```

Comment:
CI probably needs both stable and nightly since we run tests for both `c2rust-transpile` and the rustc internal dependent ones in CI.